tests/_ast/test_app.py

```diff
@@ -235,10 +236,12 @@ def _() -> None:
             _x  # type: ignore  # noqa: F821
             return
 
-        with pytest.raises(NameError) as e:
+        with pytest.raises(MarimoBaseException) as e:
             app.run()
```

Comment:
I think `app.run()` should still raise the true exception, not a wrapped one.

Reply:
Cool! I think I'm going to rename it MarimoRuntimeException though, MarimoBaseException is a little meaningless

Reply:
@dmadisetti , need any help getting this over the finish line?

Reply:
No, sorry just been busy this week. Just sat down to get this out